Lib/test/test_pathlib/test_pathlib.py

```diff
@@ -129,6 +130,7 @@ class StrSubclass(str):
         for part in p.parts:
             self.assertIs(type(part), str)
 
+    @cpython_only
```

Comment:
This doesn't fit well in this class, can you add a new test class instead?

Reply:
Done.